project/frontend/src/components/PreferenceForm.js

```diff
@@ -72,19 +104,46 @@ class PreferenceForm extends React.Component {
 
   handleSubmit(event) {
     event.preventDefault();
-    const { currLocation } = this.state;
-    currLocation['lat'] = this.props.currLocation.lat;
-    currLocation['lng'] = this.props.currLocation.lng;
-    this.setState({ currLocation });
-    console.log(this.state);
     this.props.history.push({
       pathname: '/match-results',
       state: this.state,
     });
   }
 
+  showCuisineOptions() {
+    const filter = createFilterOptions();
+    if (this.state.cuisineOptions) {
+      return (
+        <Autocomplete
+          multiple
+          name='cuisine'
+          id='cuisine'
+          options={this.state.cuisineOptions}
+          onChange={(event, newCuisineList) => {
+            this.setState({ cuisine: newCuisineList });
+          }}
+          filterOptions={(options, params) => {
+            const filtered = filter(options, params);
+            if (params.inputValue !== '' && !options.includes(params.inputValue)) {
```

Comment:
Thinking of adding something to make this case insensitive (so if I enter "mexican" and "Mexcian" is an entry in options, we won't display a custom option for the user's entry since it is the same word.